lib/cli/kit/executor.rb

```diff
@@ -9,14 +9,28 @@ def initialize(log_file:)
       end
 
       def call(command, command_name, args)
-        with_traps { with_logging { command.call(args, command_name) } }
+        with_traps do
+          with_logging do |id|
+            begin
+              command.call(args, command_name)
+            rescue => e
+              $stderr.puts "This command ran with ID: #{id}"
```

Comment:
Add a test matching the exception?

Reply:
woops, lost it in a refactor.
Added one back